### DISCUSSION TITLE
Compiling YoloV4 with Cuda >= 11 and GeForce RTX3080

### DISCUSSION THREAD

**Discussion by TidelyPom** [AlexeyAB/darknet#7359] [General]
Could anybody tell me the instruction for ARCH= in the makefile when compiling in Ubuntu 20.04 with an RTX3080 GPU and Cuda 11 ? It is not very clear from reading the readme.txt, shall I comment out the line #20 and uncomment the line 28, i.e.

ARCH= -gencode arch=compute_86,code=[sm_86,compute_86]

Or shall I comment out everything ? 

It is not clear from what I read on the page.

Thank you

**Comment by LMSAas** [comment]
yes. You should comment line 20 and uncomment line 28.
